README.md

```diff
@@ -1,5 +1,58 @@
-# ssi-dim-wallet-stub
+# SSI DIM Wallet stub application
 
-Stub application for Decentralize Identity Management (DIM) Wallet
+#### Note: *This application can not be used in production. This is designed and developed for local and integration testing only*
 
-### WIP
+The SSI Decentralize Identity Management (DIM)
+Wallet stub application is an alternative solution of wallet application to the Tractus-X.
+
+This application can be used as a temporary wallet solution for local and integration testing.
+
+This application is using Java 21 and Spring boot framework to serve REST APIs.
+
+### Run in local
```

Comment:
Besides this, also the usage via helm chart should be mentioned or linked

Reply:
Added: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/80eaae7efb3b820dacfc844a1cbb35454076d014